src/swish/io.ss

```diff
@@ -109,13 +109,15 @@
    write-osi-port
    )
   (import
-   (chezscheme)
+   (except (chezscheme) with-interrupts-disabled)
    (swish compat)
    (swish erlang)
    (swish meta)
    (swish osi)
    )
 
+  (alias with-interrupts-disabled with-interrupts-disabled-for-io)
+
```

Comment:
Would it be clearer to rename all instances of with-interrupts-disabled within this file?

Reply:
My concern had been that we might miss a case, so I had taken the more draconian approach. But I agree that it's clearer to make the use of this form explicit. We will likely adapt existing code when we add new I/O primitives and we'll follow the pattern.

I pushed a change that tries to make more deliberate use of `with-interrupts-disabled-for-io` focusing on places where we call `wait-for-io` or call something like `@close-osi-port` that itself calls `wait-for-io`.